### ISSUE TITLE
Gulp serve gives blank index.html. Installation Issue for Chromedriver while running npm install

### ISSUE THREAD

**Issue by umang-desai** [Polymer/polymer#2288]
I followed the instructions and installed the Polymer webkit dependencies. 

When i ran 'npm install && bower install', I ran into an issue. 
The installation got stuck at, 

```
"selenium install:
from: http://selenium-release.storage.googleapis.com/2.47/selenium-server-standalone-2.47.1.jar
to: /Users/home/Documents/polymer/node_modules/web-component-tester/node_modules/wct-local/node_modules/selenium-standalone/.selenium/selenium-server/2.47.1-server.jar"
---
"chrome install:
from: http://chromedriver.storage.googleapis.com/2.16/chromedriver_mac32.zip
to: /Users/home/Documents/polymer/node_modules/web-component-tester/node_modules/wct-local/node_modules/selenium-standalone/.selenium/chromedriver/2.16-x64-chromedriver"
```

Now, GULP SERVE was running but when the page opened up, I wasnt able to see anything. The page was blank. 

Please help me out here.

I wish to use Polymer to make a production level website. 

Thanks,
Umang Desai

**Comment by azakus** [comment]
What instructions did you follow?
These instructions: https://github.com/polymer/web-component-tester#running-your-tests ?

What do you mean by the installation "got stuck"?
Did npm complete, or show an error?
Do you have java installed? web-component-tester requires java to run browsers for testing.

I'm not familiar with "gulp serve". Did you run that command in Polymer?

**Comment by umang-desai** [comment]
I have java installed.
The installation did not move ahead. I waited for about 30 minutes for it
to download a 30 minute file(chromedriver).
npm did not throw an error when i ended the process (ctrl + c).

The link u mentioned (
https://github.com/polymer/web-component-tester#running-your-tests) is
after the installation phase. For the web component testers.

I followed this instructions to setup my starter kit,
https://github.com/polymerelements/polymer-starter-kit
everything worked fine till the commands, npm install && bower install (for
local dependencies).
